src/Hedgehog/Tree.fs

```diff
@@ -88,3 +88,29 @@ module Tree =
     and filterForest (f : 'a -> bool) (xs : seq<Tree<'a>>) : seq<Tree<'a>> =
         Seq.filter (f << outcome) xs
         |> Seq.map (filter f)
+
+    let rec renderList (Node (x, xs00) : Tree<string>) : List<string> =
```

Comment:
@TysonMN, is it OK if we name this `renderList`? It returns a list, at the end...

Reply:
Yep.  I don't have a preference on the name.  I only picked `renderLines` to be consistent with the names of [these Haskell Hedgehog functions](https://github.com/hedgehogqa/haskell-hedgehog/blob/7030059c089b45429c9ed23f37971e1b5334dc2e/hedgehog/src/Hedgehog/Internal/Tree.hs#L672-L719).

Reply:
Yep, those are named like that because of Monad `m` while here we're wokring with List.

I was going back and forth, but I think from a consumer's point of view `renderList` is easier to understand.

We can surely change this until we do a release.

Reply:
I do think `renderList` is better